static/src/javascripts/projects/commercial/modules/dfp/render-advert-label.js

```diff
@@ -10,12 +10,31 @@ const shouldRenderLabel = adSlotNode =>
         adSlotNode.getElementsByClassName('ad-slot__label').length
     );
 
+const createAdCloseDiv = (): HTMLDivElement => {
+    const closeDiv: HTMLDivElement = document.createElement('div');
+    closeDiv.className = 'mobilesticky-closer';
+    closeDiv.innerHTML =
+        '<svg viewbox="0 0 40 40"><path class="mobilesticky_close-x" d="M 10,10 L 30,30 M 30,10 L 10,30" /></svg>';
+    closeDiv.onclick = function onclickMobileStickyCloser() {
+        this.closest('.mobilesticky-container').remove();
```

Comment:
Is `closest` polyfilled for IE?

Reply:
Actually, it's fine in edge and I don't think we support IE anymore (?) so we're probably right as rain either way.

Reply:
Double checked both browsers both work as expected.